README.md

```diff
@@ -4,179 +4,19 @@
 [![Crates.io](https://img.shields.io/crates/v/palette.svg)](https://crates.io/crates/palette/)
 [![Docs.rs](https://docs.rs/palette/badge.svg)](https://docs.rs/palette)
 
-A Rust library that makes linear color calculations and conversion easy and accessible for anyone. It provides both precision tools that lets you work in exactly the color space you want to, as well as a general color type that abstracts away some of the technical details.
+A color management and conversion library that focuses on maintaining correctness, flexibility and ease of use. It makes use of the type system to prevent mistakes and to support a wide range of color spaces, including user defined variants and different ways to integrate it with other libraries.
```

Comment:
Reading this again, I think the second sentence could be tweaked slightly.

```suggestion
A color management and conversion library that focuses on maintaining correctness, flexibility and ease of use. It makes use of the type system to prevent mistakes, support a wide range of color spaces including user defined variants and offer different ways of integrating with other libraries.
```